xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -483,9 +532,18 @@ The `GetDouble` method throws an exception if the entire UTF-8 encoded token val
       <Docs>
         <summary>Reads the next JSON token value from the source and parses it to a <see cref="T:System.Guid" />.</summary>
         <returns>The GUID value, if the entire UTF-8 encoded token value can be successfully parsed.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The `GetGuid` method throws an exception if the entire UTF-8 encoded token value cannnot be parsed to a <xref:System.Guid> value.
```

Comment:
I think this explanation should be added to the exception itself, not to the remark. Also, which of the two exceptions are you referring to?

Reply:
I'm seeing a few similar instances of this remark about the exception being thrown, so please consider moving their explanations there too.